### PR TITLE
VM removal safeguards (was: Add --yes...)

### PR DESCRIPTION
**Pull request by tasket**
Have `qvm-remove` prompt user unless `--yes` option specified.

### CODE REVIEW

File: qubesadmin/tools/qvm_remove.py [hunk 1]
```diff
@@ -28,14 +28,26 @@
 parser = QubesArgumentParser(description=__doc__,
                              want_app=True,
                              vmname_nargs='+')
+parser.add_argument("--yes", action="store_true", dest="no_confirm",
```
Comment (marmarek) [inline qubesadmin/tools/qvm_remove.py]: Lets name it as in other tools: --force, -f

File: qubesadmin/tools/qvm_remove.py [hunk 2]
```diff
@@ -28,14 +28,26 @@
...
+
 
 
 def main(args=None, app=None):  # pylint: disable=missing-docstring
     args = parser.parse_args(args, app=app)
-    for vm in args.domains:
-        del args.app.domains[vm.name]
+    go_ahead = ""
+    if not args.no_confirm:
+        print("This will completely remove the selected VM(s)...")
```
Comment (marmarek) [inline qubesadmin/tools/qvm_remove.py:40]: I'd repeat VMs list here.

**Review by marmarek — CHANGES_REQUESTED** [review]

File: qubesadmin/tools/qvm_remove.py (hunk 1 above)
Comment (marmarek) [reply qubesadmin/tools/qvm_remove.py]: Or at least add short version: `-y`.

**Comment by tasket** [comment]
Just thought I'd mention qvm_template_postprocess uses "--really" and qvm_backup uses "--yes"
:)

**Comment by tasket** [comment]
Avoid cloning installed_by_rpm: Copying `True` status doesn't make sense here, so new VM gets default.

**Comment by marmarek** [comment]
I've pushed this from my account for travis check. Results:
 - missing --force, -f in documentation (`doc/manpages/qvm_remove.rst`). 
 - unit tests needs to be adjusted - currently wait for interactive confirmation

https://travis-ci.org/marmarek/qubes-core-admin-client/builds/325474606

**Comment by tasket** [comment]
OK, added to manpage.

**Comment by tasket** [comment]
The testing framework is unfamiliar to me. I tried looking at test modules for commands I know accept input, but I don't have time to delve into it now.

**Comment by marmarek** [comment]
I've pushed tests update. One remaining thing is pylint complaining about:
```
************* Module qubesadmin.tools.qvm_remove
R: 45, 4: Unnecessary "else" after "return" (no-else-return)
```

**Comment by tasket** [comment]
Ah, OK.

Also, review request (add vm listing) is also satisfied, I think.

**Comment by tasket** [comment]
I did a `pull` to get your commit, and then a `push`. What do you suggest?

**Comment by marmarek** [comment]
> I did a pull to get your commit, and then a push. What do you suggest?

If you want to avoid merge commit, you could do that either in opposite order, or use `pull --rebase`.

**Review by marmarek — APPROVED** [review]

**Pull request merged**